src/database/repository/user.repository.ts

```diff
@@ -9,6 +9,31 @@ class UserRepository extends Repository<User> {
   public findByEmail(email: string) {
     return this.findOne({ email });
   }
+
+  public async createUser({
```

Comment:
타입 직접 선언하지 말구 entity/user.ts에 있는 User 타입을 쓰면 더 코드가 깔끔해질거 같아!
```suggestion
  public async createUser(user: User) {..
```

Reply:
고민했었는데 ㅋㅋ 아무튼 수정할게